### DISCUSSION TITLE
Issues after mediator deployment

### DISCUSSION THREAD

**Discussion by sjkarthik** [opencrvs/opencrvs-core#6597] [Technical Q&A]
### OpenCRVS Core version

1.4.0

### OpenCRVS Countryconfig version

1.4.0

### Your operating system and version

Ubuntu 22.04

### Your node version

v16.20.0

### Which of these describes your project best?

- [X] Testing OpenCRVS for business development, educational purposes or for personal interest.
- [ ] A research project to evaluate, configure and test OpenCRVS as a possible solution for a government, an active brief or proposal.
- [ ] Actively installing and configuring OpenCRVS Core for production use in a country.
- [ ] Customising OpenCRVS Core code in production in a country

**Comment by sjkarthik** [comment]
Mediator is deployed successfully. but now callback url is giving connection timeout  after registration.  Used private/public IP and port as callback url. Could you please confirm this. Do we need anything extra to be added.
Getting below error in opencrvs/ocrvs-webhooks container. Request is not reaching to mosip.
![image](https://github.com/opencrvs/opencrvs-core/assets/58928645/7e6522a9-be2e-4773-9241-bd61ebb7b659)

**Reply by naftis** [reply]
Hi! I am happy to help you with the MOSIP integration. Are you running the [OpenCRVS-side mediator](https://github.com/opencrvs/mosip-mediator) and running it within OpenCRVS Docker Swarm or MOSIP's Kubernetes? If the former, we've recently done an example Docker compose deployment script [here](https://github.com/opencrvs/opencrvs-mosip/blob/develop/infrastructure/docker-compose.development-deploy.yml#L134), which might help you. If it's deployed within OpenCRVS's swarm, the outputs of `docker service ls` and `docker service inspect opencrvs_mosip-mediator` could be helpful for debugging.

Please blur any sensitive details if you decide to send the debug information.
